### ISSUE TITLE
Buggy volume change

### ISSUE THREAD

**Issue by michalk-k** [theonion/videojs-vast-plugin#76]
by volume change I mean setting it by .volume(X) method.
By buggy I mean video is played with 100% volume level, even if set lower. Worth to add that player UI still shows volume set to desired value.

I simply modified your example, just replacing mute method by volume(0.2)
Everything works fine with use of video and preroll provided by the example.

But then I started to play with my streams and had to spent 2 days to get what's going on.
I figured out that in my case, VAST provider provides preroll in flv format. Then I verified a few testcases and figured out that volume doesn't work as expected (is ignored) if preroll is not the same video format as video to be played after.

In short:
1.volume works while:
- preroll - flv
- video - flv

2.volume doesn't work while:
* preroll - flv
* video - mp4

3.volume doesn't work while:
* preroll - mp4
* video - flv

It's very likely depended on videojs issue reported here: https://github.com/videojs/video.js/issues/2066

I played a bit with playerjs-vast code and found, that adding setting volume in vast-preroll-ready delegate, just before player.play might be a workaround.

With regards
